### DISCUSSION TITLE
Upgrade to v18 failed following wrong steps and now not working

### DISCUSSION THREAD

**Discussion by shariq-azim** [awawa-dev/HyperHDR#613] [Q&A]
Hi there .
I had been running v17 following the tutorials from https://www.raspberrypi.com/tutorials/raspberry-pi-tv-ambient-lighting/
now i realised there were some issues with the sync of the led, in fact they were running on its own and saw upgrade is available.
So i first without removing tried installing

1 wget https://github.com/awawa-dev/HyperHDR/releases/download/v18.0.0.0/HyperHDR-18.0.0.0-Linux-`uname -m`.deb
2 sudo apt install ./HyperHDR-18.0.0.0-Linux-`uname -m`.deb
3 sudo sed -i '/^User/d' /etc/systemd/system/hyperhdr\@.serviceD (notice the typos)

and then i stopped midway to remove 

sudo systemctl stop hyperhdr@pi.service
sudo systemctl disable hyperhdr@pi

sudo apt remove hyperhdr

And again started the steps of 1,2,3,
sudo sed -i '/^User/d' /etc/systemd/system/hyperhdr\@.service
sudo systemctl daemon-reload
sudo service hyperhdr@pi restart
sudo service hyperhdr@pi status

now i see error like 
● hyperhdr@pi.service - HyperHdr ambient light systemd service  for user pi
   Loaded: loaded (/etc/systemd/system/hyperhdr@.service; disabled; vendor pres
   Active: activating (auto-restart) (Result: exit-code) since Tue 2023-07-25 1
  Process: 1469 ExecStart=/usr/bin/hyperhdr (code=exited, status=1/FAILURE)
 Main PID: 1469 (code=exited, status=1/FAILURE)

now even the led arent glowing
Please help :(

**Comment by awawa-dev** [comment]
Probably you have old, not updated OS (based on Debian Buster) and V18/V19 requires Raspberry Pi OS based on Debian Bullseye. First make a backup of your HyperHDR. Usually it can be done from web interface, but the service is not working so SSH must be used. Location of configuration depends on the user who runs HyperHDR service, default it's /home/pi/.hyperhdr (may be hidden) BTW v18 is not supported, latest version is V19.

**Reply by shariq-azim** [reply]
Understood, could be , but now how to make it working? im clueless and panicking

**Reply by awawa-dev** [reply]
As I said, make a backup first.
